ui/desktop/app/styles/app.scss

```diff
@@ -662,3 +662,31 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   gap: 0.5rem;
   width: inherit;
 }
+
+// Settings
+.settings-card {
+  .hds-card__container {
+    padding: 1rem;
+  }
+
+  .header-and-icon {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+  }
+
+  .content {
+    padding-left: 2.2rem;
+
+    .hds-form-field--layout-vertical {
+      margin-top: 2rem;
+    }
+  }
+
+  .app-information {
+    display: grid;
```

Comment:
Should this be a grid specifically for 3 columns? I'm wondering if it makes sense to use flex instead? How were you planning on doing the other cards that only have 1 or 2 columns?

Reply:
you are right, grid won't work in that case, changed to flex